nntrainer/src/bn_layer.cpp

```diff
@@ -37,10 +38,10 @@ int BatchNormalizationLayer::initialize(bool last) {
   dim = input_dim;
   output_dim = dim;
 
-  this->gamma = Tensor(dim.channel(), dim.batch(), dim.width());
-  this->beta = Tensor(dim.channel(), dim.batch(), dim.width());
-  beta.setZero();
-  gamma.setZero();
+  this->mu = Tensor(dim.channel(), dim.height(), dim.width());
```

Comment:
create a temp dim with batch_size = 1 and use it here?

Reply:
or infact does dim need to keep the batch dimension?

Reply:
Tensor constructor delegates dimension to 1 when it is omitted. So IMO it is okay.

Reply:
> does dim need to keep the batch dimension?

I think it is good idea to use `dim` to represent those dimension though. I'll make a patch :)